includes/functions.php

```diff
@@ -183,23 +183,35 @@ function dokan_redirect_if_not_seller( $redirect = '' ) {
  *
  * @param string $post_type
  * @param int    $user_id
+ * @param array  $exclude_product_types The product types that will be excluded from count
  *
  * @return array
  */
-function dokan_count_posts( $post_type, $user_id ) {
+function dokan_count_posts( $post_type, $user_id, $exclude_product_types = array( 'booking' ) ) {
     global $wpdb;
 
-    $cache_group = 'dokan_seller_product_data_' . $user_id;
-    $cache_key   = 'dokan-count-' . $post_type . '-' . $user_id;
-    $counts      = wp_cache_get( $cache_key, $cache_group );
-
+    $exclude_product_types      = esc_sql( $exclude_product_types );
```

Comment:
if you are using cache, you need to delete cache.

Reply:
Code changed as per your instruction.